src/securejoin.rs

```diff
@@ -601,7 +606,13 @@ pub(crate) fn handle_securejoin_handshake(
 
             let vg_expect_encrypted = if join_vg {
                 let group_id = get_qr_attr!(context, text2).to_string();
-                let (_, is_verified_group, _) = chat::get_chat_id_by_grpid(context, group_id);
+                // This is buggy, is_verified_group will always be
```

Comment:
i think in a next pr, we should remove this whole vg_expect_encrypted thingie.

Reply:
yep, more cleanup is needed here.  but i'm picking my battles and keeping logic changes to a minimum here for now.

Reply:
yip, you're right. it makes totally sense to be super-careful here.